titiler/pgstac/db.py

```diff
@@ -9,7 +9,11 @@
 
 
 async def connect_to_db(
-    app: FastAPI, settings: Optional[PostgresSettings] = None
+    app: FastAPI,
+    settings: Optional[PostgresSettings] = None,
+    kwargs: Dict[str, Any] = {
+        "options": "-c search_path=pgstac,public -c application_name=pgstac"
+    },
```

Comment:
We should avoid having a dict as default in the parameters, maybe do something like 

```python
async def connect_to_db(
    app: FastAPI,
    settings: Optional[PostgresSettings] = None,
    pool_kwargs: Optional[Dict[str, Any]] = None,
) -> None:
    pool_kwargs = pool_kwargs if pool_kwargs is not None else {
        "options": "-c search_path=pgstac,public -c application_name=pgstac"
    }
```

Reply:
That looks good!